eo-parser/src/test/resources/org/eolang/parser/packs/syntax/text-block.yaml

```diff
@@ -0,0 +1,38 @@
+xsls: []
+tests:
+  - //o[@base='string' and not(text())]
+  - //o[@base='string' and text()='simple']
+  - //o[@base='string' and normalize-space(text())='Hi, there Adiós']
+  - //o[@base='string' and normalize-space(text())='first second']
+  - //o[@base='string' and text()='   third']
+  - //o[@base='string' and text()='forth']
+eo: |
+  # Empty
+  """""" > txt
+
+  # Single line
+  """simple""" > txt
```

Comment:
@andreoss I suggest we fail in this situation. The marker `"""` should _always_ be trailed by EOL.

Reply:
@yegor256 Done.